Cards/Include/Card.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once

```

Comment:
Card, MonsterCard oraz SpellCard to powinny być trzy osobne pliki. Jeden plik = jedna klasa.